TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedDocumentNodes.html

```diff
@@ -0,0 +1,46 @@
+{namespace neos=TYPO3\Neos\ViewHelpers}
+<f:layout name="EditImage"/>
+
+<f:section name="Title">RelatedDocumentNodes view of AssetController</f:section>
+
+<f:section name="Content">
+      <legend>{asset.resource.filename}</legend>
+
+      <table class="neos-info-table">
+        <thead>
+          <tr>
+            <th>{neos:backend.translate(id: 'media.relatedDocumentNodes.document', source: 'Modules', package: 'TYPO3.Neos')}</th>
+            <th>{neos:backend.translate(id: 'media.relatedDocumentNodes.reference', source: 'Modules', package: 'TYPO3.Neos')}</th>
+            <th class="neos-aRight">{neos:backend.translate(id: 'media.field.lastModified', source: 'Modules', package: 'TYPO3.Neos')}</th>
+          </tr>
+        </thead>
+        <tbody>
+          <f:for each="{relatedDocumentNodes}" as="relatedDocumentNode">
```

Comment:
I'd suggest to improve this a little
![image](https://cloud.githubusercontent.com/assets/903567/14164105/ee2741f8-f758-11e5-89a6-6c0e60dd67fe.png)